tests/core/dom/element/scrolls.js

```diff
@@ -0,0 +1,118 @@
+/* bender-tags: editor */
+
+( function() {
+	'use strict';
+	bender.editor = true;
+	bender.test( {
+		_assertBothScrolls: function( expected, actual ) {
+			assert.areSame( expected.x, actual.scrollLeft );
+			assert.areSame( expected.y, actual.scrollTop );
+		},
+
+		'test getDocumentScrollPosition': function() {
+			var element = document.getElementById( 'small' ),
+				ckEl = new CKEDITOR.dom.element( element ),
+				doc,
+				docEl;
+			// Reset position with native methods
+			doc = ckEl.getDocument();
+			docEl = doc.$.scrollingElement || doc.$.documentElement || doc.$.body;
+			docEl.scrollTop = 0;
+			docEl.scrollLeft = 0;
+			ckEl.$.scrollTop = 0;
+			ckEl.$.scrollLeft = 0;
+
+			// document no-scroll, element no-scroll
+			this._assertBothScrolls( { x: 0, y: 0 }, ckEl.getDocumentScrollPosition() );
+
+			// document no-scroll, element scroll
+			ckEl.$.scrollTop = 123;
+			ckEl.$.scrollLeft = 123;
+			this._assertBothScrolls( { x: 0, y: 0 }, ckEl.getDocumentScrollPosition() );
+
+			// document scroll, element scroll
+			docEl.scrollTop = 20;
+			docEl.scrollLeft = 20;
+			this._assertBothScrolls( { x: 20, y: 20 }, ckEl.getDocumentScrollPosition() );
+
+			// document scroll, element no-scroll
+			ckEl.$.scrollTop = 0;
+			ckEl.$.scrollLeft = 0;
+			this._assertBothScrolls( { x: 20, y: 20 }, ckEl.getDocumentScrollPosition() );
+		},
+
+		'test getScrollPosition': function() {
```

Comment:
This test is very similar to the previous one and it seems that only the checked element is different. Maybe it'd be better to create factory for such tests?